### PR TITLE
Remove initialize back

### PR DESCRIPTION
**Pull request by muki**
I made a bit of a mess with branching off of branches, but this should now all be working and we are done depending on initializeBack.js

@mzgajner pls just do a sanity check, there are no errors thrown on my end, and then pull the trigger

### CODE REVIEW

File: cards/_components/Card/Embed.vue [hunk 1]
```diff
@@ -30,13 +30,46 @@
...
+    copyEmbedCode() {
+      this.$refs.embedInput.select();
+      let succeed = false;
+      try {
+          succeed = document.execCommand("copy");
+      } catch (e) {
+          return e;
+      }
+      this.$refs.embedInput.blur();
+      this.$refs.copyButton.textContent = 'SKOPIRANO!'
```
Comment (mzgajner) [inline cards/_components/Card/Embed.vue]: I wouldn't manipulate the DOM here, you could just grab the text directly from `this.embedCode` and then set something like `this.copied = true` and update the button text in the template based on that.

File: cards/_components/Card/Share.vue [hunk 2]
```diff
@@ -13,27 +13,54 @@
...
+      let succeed = false;
+      try {
+          succeed = document.execCommand("copy");
+      } catch (e) {
+          return e;
+      }
+
+      // change text
+      if (succeed) {
+        this.$refs.copyButton.textContent = "SKOPIRANO!";
```
Comment (mzgajner) [inline cards/_components/Card/Share.vue]: Same issue as in the embed.

File: cards/_components/Card/Embed.vue [hunk 3]
```diff
@@ -30,13 +30,46 @@
...
 export default {
   name: 'CardEmbed',
+
   computed: {
     embedCode() {
-      return `&#x3C;script&#x3E;(function(d,script){script=d.createElement(&#x27;script&#x27;);script.type=&#x27;text/javascript&#x27;;script.async=true;script.onload=function(){iFrameResize({log:true,checkOrigin:false})};script.src=&#x27;https://cdn.parlameter.si/v1/parlassets/js/iframeResizer.min.js&#x27;;d.getElementsByTagName(&#x27;head&#x27;)[0].appendChild(script);}(document));&#x3C;/script&#x3E;&#x3C;iframe frameborder=&#x22;0&#x22; width=&#x22;100%&#x22; src=&#x22;${this.url}&#x26;embed=true&#x22;&#x3E;&#x3C;/iframe&#x3E;`;
+      let newUrl = this.url;
+      if (!this.refresh) {
+        const todaysdate = new Date;
+        const today = '' + todaysdate.getDay() + '.' + todaysdate.getMonth() + '.' + todaysdate.getFullYear();
```
Comment (mzgajner) [inline cards/_components/Card/Embed.vue]: Use template literals here. Also, I think we use [`date-fns`](https://date-fns.org/) to manipulate/format dates in other places, maybe take a look at that.

**Review by mzgajner — CHANGES_REQUESTED** [review]
Good job getting rid of the jQuery stuff, that was on the bucket list for a while! :+1:

Added some minor change requests, then it can be merged.

**Comment by muki** [comment]
All requested changes implemented. I think. :)

**Pull request merged**